src/adtfhir_to_opal/compose.yaml

```
@@ -4,13 +4,13 @@ services:
       context: .
       dockerfile: Dockerfile
     environment:
-      OUTPUT_FOLDER: "/opt/bitnami/spark/opal-output"
-      KAFKA_TOPIC_YEAR_SUFFIX: ""
+      OUTPUT_FOLDER: "/opt/bitnami/spark/opal-output/"
+      KAFKA_TOPIC_YEAR_SUFFIX: ".2022"
```

```suggestion
      KAFKA_TOPIC_YEAR_SUFFIX: ""
```

sollen wir das tatsächlich ändern? lokal macht es sinn, aber die Benennungen auch der anderen post-gateway topics ist schon sehr speziell

---

ja stimmt, dachte es ist egal wenn es ohnehin geändert werden msus falls andere Standorte andere naming conventions haben, aber tut mir auch nicht weh das wieder rückgängig zu machen :)